modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/components/BackButtonManagementBar.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '../../../css/content_editor/ContentEditorManagementBar.scss';
+
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import {ManagementToolbar} from 'frontend-js-components-web';
+import React from 'react';
+
+export default function BackButtonManagementBar({
+	backURL,
+	headerTitle,
+}: {
+	backURL: string;
+	headerTitle: string;
+}) {
+	return (
+		<ManagementToolbar.Container className="border content-editor__management-bar position-fixed">
```

Comment:
This component id not related to Content Editor, so if this class `content-editor__management-bar` is needed, better create a new one (maybe `cms-management-bar` 🤔 )